drivers/pds/lib/utils.go

```diff
@@ -299,10 +312,10 @@ func CheckNamespace(namespace string) (bool, error) {
 					Labels: map[string]string{pxLabel: "true"},
 				},
 			}
-			log.Infof("Creating namespace %v", namespace)
+			log.InfoD("Creating namespace %v", namespace)
 			ns, err = k8sCore.CreateNamespace(nsName)
 			if err != nil {
-				log.Errorf("Error while creating namespace %v", err)
+				//log.FailOnError(err, "Error while creating namespace")
```

Comment:
remove the commented line if not required

Reply:
done